integrations/pooltogether-v4/hooks/use-user-balance-withdraw.tsx

```diff
@@ -0,0 +1,27 @@
+import { useErc20BalanceOf, useErc20Decimals } from '@turbo-eth/erc20-wagmi'
+import { formatUnits } from 'ethers/lib/utils'
+import { useAccount, useNetwork } from 'wagmi'
+
+import { useLoadContractFromChainId } from '@/actions/pooltogether-v4/hooks/use-load-contract-from-chain-id'
+import { TICKET_CONTRACT } from '@/actions/pooltogether-v4/ticket-contract-list'
+
+export function useUserBalanceWithdraw() {
+  const { address: accountAddress } = useAccount()
+  const { chain } = useNetwork()
+
+  const address = useLoadContractFromChainId(TICKET_CONTRACT)
+  const { data: decimals } = useErc20Decimals({ address })
+
+  const {
+    data: erc20Balance,
+    isError,
+    isLoading,
+  } = useErc20BalanceOf({
+    chainId: chain?.id,
+    address,
+    watch: true,
+    args: [accountAddress as `0x${string}`],
```

Comment:
overall avoid type assertions, doing so will mask on the typescript compiler the case when `accountAddress` is `undefined`, which should be properly handled

Reply:
So how to handle it properly since most components/functions want `0x${string}` instead of  a string (e.g useErc20Decimals) ?